src/main/kotlin/simulator/lotto/Rank.kt

```diff
@@ -0,0 +1,53 @@
+package simulator.lotto
+
+enum class Rank {
+    FIRST {
+        override fun prize(): Int {
+            return 2_000_000_000
+        }
+
+        override fun match(): Int {
+           return 6
+        }
+    },
```

Comment:
```suggestion
enum class Rank(val prize: Int, val match: Int) {
    FIRST(2_000_000_000, 6),
```
함수로 구현해주셔도 좋지만, 행동이 아닌 상태를 나타내는 것이라면 위와 같이 구현하셔도 좋습니다.